truss/parsepkgname/parsepkgname_test.go

```diff
@@ -0,0 +1,89 @@
+package parsepkgname
+
+import (
+	"io"
+	"strings"
+	"testing"
+)
+
+type testScanner struct {
+	contents [][]rune
+	position int
+}
+
+func (t *testScanner) ReadUnit() ([]rune, error) {
+	if t.position < len(t.contents) {
+		rv := t.contents[t.position]
+		t.position += 1
+		return rv, nil
+	} else {
```

Comment:
You don't need an `else` here. `golint` will catch stylistic stuff like this for you.

```
➜  dummy grep -A 10 "func even" main.go
func even(i int) bool {
    if i%2 == 0 {
        return true
    } else {
        return false
    }
}
➜  dummy golint main.go
main.go:15:12: if block ends with a return statement, so drop this else and outdent its block
```
(I know explicitly returning `true` or `false` here is dum dum; it was just the first example I thought of.)